web/pgadmin/static/js/helpers/Layout/index.jsx

```diff
@@ -485,9 +492,13 @@ export default function Layout({groups, noContextGroups, getLayoutInstance, layo
         },
         restore: true,
       };
-      savedTab.metaData = updatedMetaData;
+      return {
```

Comment:
I didn't get why was this changed. Previous code looks fine to me.

Reply:
Done.